src/many-kvstore/tests/kvstore.rs

```diff
@@ -0,0 +1,75 @@
+pub mod common;
+use crate::common::{setup, Setup};
+use minicbor::bytes::ByteVec;
+
+use many::{
+    server::module::kvstore::{
+        KvStoreModuleBackend, 
+        KvStoreCommandsModuleBackend, 
+        InfoArg,
+        PutArgs,
+        GetArgs,
+        DeleteArgs
+    },
+};
+
+#[test]
+fn kvstore_info() {
+    let Setup {
+        module_impl, 
+        id, 
+        ..
+    } = setup();
+    
+    let result = module_impl.info(&id, InfoArg {});
+    assert!(result.is_ok());
+}
+
+#[test]
+fn kvstore_put() {
+    let Setup {
+        mut module_impl, 
+        id, 
+        ..
+    } = setup();
+
+    let data = PutArgs {
+        key: ByteVec::from(vec![1]),
```

Comment:
You can use `vec![1].into()` and remove the reference to `ByteVec`

Reply:
I did this however I was not able to remove the reference from the file as I needed it for the assertion down below. Please advise if this would be necessary.